### PR TITLE
Add `motion` prefix for motion tokens

### PR DESCRIPTION
**Pull request by alex-page**

### CODE REVIEW

File: .changeset/forty-mangos-judge.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-tokens': minor
+---
+
+Add motion prefix for motion tokens
```
Comment (lgriffee) [inline .changeset/forty-mangos-judge.md:5]: Is this a major since we are changing the token names?

Comment (alex-page) [reply .changeset/forty-mangos-judge.md:5]: The intention is for this to not be a breaking change as we are adding new values not removing any. The breaking change will be removing the values.

**Review by lgriffee — APPROVED** [review]

File: polaris-react/scripts/build-validate.js [hunk 2]
```diff
@@ -45,7 +45,9 @@ function validateStandardBuild() {
   assert.ok(cssContent.includes('.Polaris-Avatar {'));
   assert.ok(cssContent.includes('.Polaris-BulkActions__BulkActionButton {'));
   assert.ok(cssContent.includes('@keyframes p-keyframes-bounce {'));
-  assert.ok(cssContent.includes('--p-keyframes-bounce:p-keyframes-bounce;'));
+  assert.ok(
+    cssContent.includes('--p-motion-keyframes--bounce:p-keyframes-bounce;'),
+  );
 }
```
Comment (alex-page) [inline polaris-react/scripts/build-validate.js]: @aaronccasanova @lgriffee there are some problems with the keyframe generation with the new custom properties. I don't have a lot of context and tried a couple of things. I could use some help here either pairing or if it's not too long just ship the changes into the PR.

That is the only blocker here :)

File: polaris-react/scripts/build-validate.js [hunk 3]
```diff
@@ -45,7 +45,9 @@ function validateStandardBuild() {
   assert.ok(cssContent.includes('.Polaris-Avatar {'));
   assert.ok(cssContent.includes('.Polaris-BulkActions__BulkActionButton {'));
   assert.ok(cssContent.includes('@keyframes p-keyframes-bounce {'));
-  assert.ok(cssContent.includes('--p-keyframes-bounce:p-keyframes-bounce;'));
+  assert.ok(
+    cssContent.includes('--p-motion-keyframes--bounce:p-keyframes-bounce;'),
```
Comment (aaronccasanova) [inline polaris-react/scripts/build-validate.js]:
```suggestion
    cssContent.includes('--p-motion-keyframes-bounce:p-keyframes-bounce;'),
```

File: polaris-react/scripts/build-validate.js [hunk 4]
```diff
@@ -69,7 +71,9 @@ function validateEsNextBuild() {
   assert.ok(cssContent.includes('.Polaris-Avatar_z763p {'));
   assert.ok(cssKeyframesContent.includes('@keyframes p-keyframes-spin {'));
   assert.ok(
-    cssKeyframesContent.includes('--p-keyframes-spin:p-keyframes-spin;'),
+    cssKeyframesContent.includes(
+      '--p-motion-keyframes--spin:p-keyframes-spin;',
```
Comment (aaronccasanova) [inline polaris-react/scripts/build-validate.js]:
```suggestion
      '--p-motion-keyframes-spin:p-keyframes-spin;',
```

File: polaris-react/src/components/Backdrop/Backdrop.scss [hunk 5]
```diff
@@ -8,7 +8,8 @@
   display: block;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   background-color: rgba(0, 0, 0, 0.5);
-  animation: var(--p-keyframes-fade-in) var(--p-duration-200) 1 forwards;
+  animation: var(--p-motion-keyframes--fade-in) var(--p-motion-duration-200) 1
```
Comment (aaronccasanova) [inline polaris-react/src/components/Backdrop/Backdrop.scss:11]:
```suggestion
  animation: var(--p-motion-keyframes-fade-in) var(--p-motion-duration-200) 1
```

File: polaris-react/src/components/Indicator/Indicator.scss [hunk 6]
```diff
@@ -26,13 +26,15 @@
 .pulseIndicator::before {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   z-index: 1;
-  animation: var(--p-keyframes-bounce) var(--p-duration-5000) ease infinite;
+  animation: var(--p-motion-keyframes--bounce) var(--p-motion-duration-5000)
```
Comment (aaronccasanova) [inline polaris-react/src/components/Indicator/Indicator.scss:29]:
```suggestion
  animation: var(--p-motion-keyframes-bounce) var(--p-motion-duration-5000)
```

**Comment by aaronccasanova** [comment]
Steps from the above commit:
```sh
npx @shopify/polaris-migrator styles-replace-custom-property \
  --decl="/.+/" --from="--p-ease" --to="--p-motion-ease" "{polaris-react,polaris.shopify.com}/**/*.scss"

npx prettier --write "{polaris-react,polaris.shopify.com}/**/*.scss"

npx stylelint "{polaris-react,polaris.shopify.com}/**/*.scss"
```

**Review by aaronccasanova — APPROVED** [review]

**Pull request merged**